src/pages/ProductDetail/ProductDetail.js

```diff
@@ -71,40 +89,19 @@ function ProductDetail() {
           </div>
         </div>
       </div>
-      <OrderBar
-        id={id}
-        names={names}
-        price={price}
-        sub_description={sub_description}
-        image_url={image_url}
-        detail={detail}
-        descriptions={descriptions}
-      />
+      {detailData.id && (
+        <OrderBar
+          id={detailData.id}
+          names={detailData.names}
+          price={detailData.price}
+          sub_description={detailData.sub_description}
+          image_url={detailData.image_url}
+          detail={detailData.detail}
+          descriptions={detailData.descriptions}
```

Comment:
```suggestion
          {...detailData}
```
state의 모든 값을 넘겨줄 때는 이와 같은 방식으로 한번에 넘겨줄 수 있겠습니다!